One-Electron-Property/IR-Intensities/intensities.py

```diff
@@ -0,0 +1,173 @@
+#
+# Conversion factor for taking IR intensities from ((D/A)^2 / amu) to a.u. ((e a0 / a0)^2 / me):
+conv_kmmol = (1 / conv_dip_grad_au2DAamu) ** 2
+# Multiply by (1 / (4 pi eps_0)) in a.u. (1 / (e^2 / a0 Eh)) to get to units of (Eh a0 / me)
+# Multiply by (Na pi / 3 c^2) in a.u. (Na = mol^-1; c = Eh / alpha^2 me) to get to units of a0/mol
+conv_kmmol *= psi_na * np.pi * (1 / 3) * psi_alpha**2
+# Convert to units of km/mol
+conv_kmmol *= psi_bohr2m * (1 / 1000)
+#
+conv_ir_DAamu2kmmol = conv_kmmol                                                                                # (D/A)^2/amu -> km/mol
+
+
+
+mol = psi4.geometry("""
+C  -1.2602962432       0.0000000000       0.0000000000
+C   1.2602962432       0.0000000000       0.0000000000
+H  -2.3303912628      -1.7258024530       0.0000000000
+H   2.3303912628      -1.7258024530       0.0000000000
+H  -2.3303912628       1.7258024530       0.0000000000
+H   2.3303912628       1.7258024530       0.0000000000
+
+units bohr
+symmetry c1
+""")
+
+psi4.set_options({'basis': 'dz',
+                  'scf_type': 'direct',
+                  'd_convergence': 9,
+                  'e_convergence': 11,
+                 })
+
+# Run SCF frequency calculation for testing
+e, wfn = psi4.frequencies('scf', return_wfn=True)
+
+# Relevant Variables
+natoms = mol.natom()
+
+# Get array of atomic masses in a.u.
+masses = np.array([mol.mass(i) * (1 / psi_au2amu) for i in range(mol.natom())])
+
+# Get Hessian from wavefunction
+Hess = np.asarray(wfn.hessian())
+print("\nMolecular Hessian (a.u.):\n", Hess)
+
+# Mass weight Hessian
+M = np.diag(1 / np.sqrt(np.repeat(masses, 3))) # Matrix for mass weighting
+mH = M.T.dot(Hess).dot(M) 
+#print("\nMass-weighted Hessian (a.u.):\n", mH)
+#print("\nMass-weighted Hessian (hartree/(bohr^2 amu)):\n", mH * (1 / psi_au2amu))
+
+# Mass-weighted normal modes from Hessian
+k2, Lxm = np.linalg.eigh(mH) 
+#print("\nEigenvalues (a.u.):\n", k2) 
+#print("\nEigenvectors (unitless):\n", Lxm) 
+print("\nEigenvalues (hartree/(bohr^2 amu)):\n", k2 * (1 / psi_au2amu)) 
+#print("\nEigenvalues (J/kg*m^2 = s^-2):\n", k2 * (1 / psi_au2amu) * hartree2joule / (sqbohr2sqmeter * amu2kg)) 
+
+normal_modes = []
+mode = 3 * natoms - 1
+print("\nNormal Modes (cm^-1):")
+while mode >= 6:
+    if k2[mode] >= 0.0:
+        normal_modes.append(np.sqrt(k2[mode]))
+        print("%.2f" % (np.sqrt(k2[mode]) * conv_freq_au2wavenumbers))
+    else:
+        normal_modes.append(np.sqrt(abs(k2[mode])))
+        print("%.2fi" % (np.sqrt(abs(k2[mode])) * conv_freq_au2wavenumbers))
+    mode -= 1
+
+# Test frequencies
+psi_freq = wfn.frequency_analysis['omega'].data[6:]
+psi_freq = np.flip(psi_freq, 0)
+for i in range(len(normal_modes)):
+    psi4.compare_values(psi_freq[i], normal_modes[i] * conv_freq_au2wavenumbers, 6, "FREQ-TEST")
+
+# Read in dipole derivatives
+dipder = wfn.variables().get("CURRENT DIPOLE GRADIENT", None)
+dipder = np.asarray(dipder)
+#print("\nDipole Derivatives (a.u.):\n", dipder)
```

Comment:
Remove this.

Reply:
"this" == print statement? Or "this" == unnecessary comment

Reply:
Print statement.